issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListSpinnerAdapter.kt

```diff
@@ -45,6 +45,7 @@ class IssuerListSpinnerAdapter internal constructor(
             binding = SpinnerListWithImageBinding.inflate(LayoutInflater.from(context), parent, false)
             view = binding.root
             viewHolder = IssuerListSpinnerViewHolder(binding, imageLoader, paymentMethod, hideIssuerLogo)
+            view.tag = viewHolder
```

Comment:
Can you make sure that this crash doesn't happen somewhere else? As I guess there's some other places that is done the same way this class is done

Reply:
I did, this is the only place we missed adding the tag. Thanks for pointing out!